packages/desktop-client/src/components/App.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
This file has quite a lot of functional changes (you're porting it from a class component to a hook). Any chance you could do this as a separate PR so we can carefully review the changes?

Reply:
Create a separate PR for this: https://github.com/actualbudget/actual/pull/1535